### PR TITLE
promote registration-operator to 2.3

### PR DESCRIPTION
**Pull request by zhiweiyin318**

### CODE REVIEW

File: ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-release-2.3-postsubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        - osci/publish
+        command:
+        - make
+        env:
+        - name: COMPONENT_NAME
+          value: registration-operator
+        - name: COMPONENT_VERSION
+          value: 0.1.0
+        - name: OSCI_IMAGE_REMOTE_REPO_SRC
+          value: registry.svc.ci.openshift.org/open-cluster-management
```
Comment (qiujian16) [inline ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-release-2.3-postsubmits.yaml:77]: I think we should change this

File: ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-release-2.3-postsubmits.yaml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    branches:
+    - ^release-2\.3$
+    cluster: build02
+    decorate: true
+    name: branch-ci-open-cluster-management-registration-operator-release-2.3-publish
+    spec:
+      containers:
+      - args:
+        - -f
+        - /opt/build-harness/Makefile.build-harness-openshift-ci
```
Comment (qiujian16) [inline ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-release-2.3-postsubmits.yaml:67]: change to makefile.prow

File: ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-release-2.3-postsubmits.yaml [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        - make
+        env:
+        - name: COMPONENT_NAME
+          value: registration-operator
+        - name: COMPONENT_VERSION
+          value: 0.1.0
+        - name: OSCI_IMAGE_REMOTE_REPO_SRC
+          value: registry.svc.ci.openshift.org/open-cluster-management
+        - name: SELF
+          value: make -f /opt/build-harness/Makefile.build-harness-openshift-ci
```
Comment (qiujian16) [inline ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-release-2.3-postsubmits.yaml:79]: makefile.prow

File: ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-release-2.3-postsubmits.yaml (hunk 1 above)
Comment (zhiweiyin318) [reply ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-release-2.3-postsubmits.yaml]: done

File: ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-release-2.3-postsubmits.yaml (hunk 2 above)
Comment (zhiweiyin318) [reply ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-release-2.3-postsubmits.yaml]: done

File: ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-release-2.3-postsubmits.yaml (hunk 3 above)
Comment (zhiweiyin318) [reply ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-release-2.3-postsubmits.yaml]: done

**Pull request merged**